src/status_im/ui/screens/contacts/events.cljs

```diff
@@ -278,10 +278,12 @@
                                                         (assoc :bot id
                                                                :type :command))]))
                                (into {}))
-          contacts (into {} contacts-list)]
-      {:db         (assoc db :contacts/contacts contacts
+          contacts (into {} contacts-list)
+          ;;TODO temporary hide wallet contact, this code should be deleted after wallet contact will be deleted
+          contacts' (assoc-in contacts ["wallet" :pending?] true)]
```

Comment:
Same here, this variable name is not descriptive.

Reply:
same here